.gitmodules

```
@@ -1,4 +1,4 @@
 [submodule "tests/Engine/EngineTests/EngineTestData"]
 	path = tests/Engine/EngineTests/EngineTestData
 	url = git@github.com:Flagsmith/engine-test-data.git
-	branch = v1.0.0
+	tag = v2.3.0
```

Let's include the variant sorting tests:

```suggestion
	tag = v2.4.0
```